app/controllers/api/v1/merchants/merchant_invoices_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::Merchants::MerchantInvoicesController < ApplicationController
+
+	def index
+		render json: Merchant.find(params[:id]).invoices
```

Comment:
@Dpalazzari what's your thought on two-lining this to
```
merchant = Merchant.find(params[:id])
render json: merchant.invoices
```
?

More verbose, but maybe more readable?